sql/core/src/test/resources/sql-tests/inputs/literals.sql

```diff
@@ -153,3 +153,12 @@ select interval '1' year to second;
 select '1' year to second;
 select interval 1 year '2-1' year to month;
 select 1 year '2-1' year to month;
+SET spark.sql.ansi.enabled=false;
```

Comment:
not related to this PR, but it's really annoying to duplicate the test cases with ansi mode on and off.

I'm wondering if we can introduce a `-- import abc.sql` directive so that we can easily import test cases in `ansi/interval.sql`, with several ansi specific test cases. cc @maropu

Reply:
Ah, I see. That's certainly annoying. I'll try to make a pr later for that approach.